### ISSUE TITLE
[Question] Can camoufox browser be opened in parallel like we can do using selenium grid?

### ISSUE THREAD

**Issue by juanfrilla** [daijro/camoufox#143]
Can camoufox be opened in parallel like selenium grid? If so, is there anything to configure the opened instances? 
If not, are you planning to implement this feature?

Regards

**Comment by JWriter20** [comment]
@juanfrilla Yes. Each `Camoufox(...)` (or `AsyncCamoufox(...)`) is a fully independent browser process with its own profile and its own auto-generated fingerprint, so you can run as many in parallel as your hardware allows — just launch them from separate threads, async tasks, or processes. For a Selenium-Grid-style setup, use server mode: call `launch_server(...)` (or the `camoufox server` CLI) to expose a WebSocket endpoint, then `firefox.connect(ws_endpoint)` to it from multiple clients. There's nothing special to configure out of the box; if you want isolated persistent profiles, just give each instance its own `user_data_dir`.
